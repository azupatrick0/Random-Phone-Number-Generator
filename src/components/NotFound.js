import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <>
    <div style={{ marginTop: '40vh', textAlign: 'center', fontSize: 50, fontWeight: 'bold', fontFamilt: 'Arial', color: 'red' }}>
      Page Not Found
    </div>
    <div style={{ marginTop: '2vh', textAlign: 'center', fontSize: 20, fontWeight: 'bold', fontFamilt: 'Arial', color: 'green' }}><Link to='/'>Back to home</Link></div>
  </>
);

export default NotFound;
