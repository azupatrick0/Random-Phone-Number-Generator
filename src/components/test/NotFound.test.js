import React from 'react';
import NotFound from '../NotFound';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';

configure({ adapter: new Adapter() });

it('renders NotFound component without crashing', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
