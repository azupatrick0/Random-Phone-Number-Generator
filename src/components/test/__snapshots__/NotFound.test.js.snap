// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFound component without crashing 1`] = `ShallowWrapper {}`;
